requirements-test.txt

```diff
@@ -5,7 +5,11 @@
 # Kali Linux, so it makes sense to force the installation of Ansible
 # 2.10 or newer.
 ansible>=2.10,<3
-ansible-lint
+# ansible-lint has a number of breaking changes with the update to v5 as we have
+# documented in https://github.com/cisagov/skeleton-ansible-role/issues/69
+# We can unpin once everything with the v5 update has shaken out and we have a
+# clear path forward.
+ansible-lint<5
 boto3
```

Comment:
Does this role need `boto3`? I don't see a [cisagov/ansible-role-cyhy-core](https://github.com/cisagov/ansible-role-cyhy-core) requirement or any `lookup()` uses that would require it at a glance.